### ISSUE TITLE
Padding and Embedding 

### ISSUE THREAD

**Issue by BraidenKirkland** [RoyLQ/Advanced_AutoCodeCompletion#27]
[https://www.tensorflow.org/guide/keras/masking_and_padding](https://www.tensorflow.org/guide/keras/masking_and_padding)

Going off of what I read in this article, it looks like they perform the padding and embedding steps back to back. In the embedding step, they use `mask_zero=True` to indicate that the zero's represent a mask. Maybe this is something we could handle in `word_embedding.py`?

Also, if we create the embedding table from the sequence list in one shot, then maybe we do not have to feed both of them as input into the LSTM. I will keep reading into this further.
